tests/runtest.proj

```diff
@@ -379,6 +379,26 @@ namespace $([System.String]::Copy($(Category)).Replace(".","_").Replace("\","").
     <MSBuild Projects="$(MSBuildProjectFile)" Targets="UploadPerfAssemblies" />  
   </Target>  
 
+  <PropertyGroup>
+    <CMakeCache>$(RootBinDir)obj\$(BuildOS).$(BuildArch).$(BuildType)\CMakeCache.txt</CMakeCache>
+    <CMakeCacheContent>$([System.IO.File]::ReadAllText($(CMakeCache)))</CMakeCacheContent>
+    <PGODisabledAtBuild>$([System.Text.RegularExpressions.Regex]::Match($(CMakeCacheContent), `CLR_CMAKE_PGO_OPTIMIZE:\S+=(0|1)`).Value.EndsWith("0"))</PGODisabledAtBuild>
+
+    <RunPGOCheck>true</RunPGOCheck>
+    <RunPGOCheck Condition="$(Configuration) != 'Release' OR $(RunningOnUnix) == 'true'">false</RunPGOCheck>
+    <RunPGOCheck Condition="$(__BuildArch) == 'arm' OR $(__BuildArch) == 'arm64'">false</RunPGOCheck>
+    <RunPGOCheck Condition="$(PGODisabledAtBuild) == 'true'">false</RunPGOCheck>
+  </PropertyGroup>
+
+  <Target Name="CheckForPGO">
+    <ItemGroup>
+      <PGOEnabled Include="$(CoreCLRBinDir)\coreclr.dll" />
+      <PGOEnabled Include="$(CoreCLRBinDir)\clrjit.dll" />
+    </ItemGroup>
+
+    <Message Text="Checking if the following DLLs are properly compiled with PGO" Importance="High" />
+    <Exec  Command="python.exe $(MSBuildThisFileDirectory)scripts/pgocheck/pgocheck.py @(PGOEnabled)" />
```

Comment:
I recommend using just `python` instead of `python.exe` in the command; that way it should just work on Linux too when we port support over there